src/hooks/useCustomControl.tsx

```diff
@@ -13,15 +13,15 @@ function useCustomControl({ data: value, errors, handleChange, path, schema }: C
 	}, [path]);
 
 	const onBlur = useCallback(
-		(e) => {
+		(e: any) => {
```

Comment:
any ?

Reply:
I had a go at guessing a type but it's variant. I decided to unblock myself and accept already existent technical debt.